docker-compose.yml

```diff
@@ -5,8 +5,7 @@ services:
       context: ./
       dockerfile: Dockerfile
     container_name: woeip.app
-    # TODO Update this with the appropriate prefix
-    image: woeip:latest
+    image: openoakland/woeip:latest
```

Comment:
Similar to above, correct the Docker image name: https://hub.docker.com/r/openoakland/woeip/.